### PR TITLE
Solidus 3 preparation

### PR DESCRIPTION
**Pull request by cpfergus1**
This PR prepares the extension to work on Solidus 3.0 by removing Solidus 2.11 deprecations that would become errors on 3.0.
This satisfies issue [#34](https://github.com/solidusio-contrib/solidus_log_viewer/issues/34)

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
 Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
 
 # Requires factories defined in lib/solidus_log_viewer/factories.rb
```
Comment (kennyadsl) [inline spec/spec_helper.rb:18]: Do we need to update this comment as well?

**Review by kennyadsl — CHANGES_REQUESTED** [review]

File: spec/spec_helper.rb (hunk 1 above)
Comment (cpfergus1) [reply spec/spec_helper.rb:18]: I believe the makes perfect sense!

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -15,8 +15,10 @@
 # in spec/support/ and its subdirectories.
 Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
 
-# Requires factories defined in lib/solidus_log_viewer/factories.rb
-require 'solidus_log_viewer/factories'
+# Will load Solidus core factory first and then the ones
+# defined in `lib/solidus_log_viewer/testing_support/factories`.
+
```
Comment (kennyadsl) [inline spec/spec_helper.rb:20]:
```suggestion
```

**Review by kennyadsl — CHANGES_REQUESTED** [review]
Nice, I'd just remove the space between the comment and the code it describes.

File: spec/spec_helper.rb (hunk 2 above)
Comment (cpfergus1) [reply spec/spec_helper.rb]: patched!

**Review by kennyadsl — APPROVED** [review]

**Pull request merged**